### DISCUSSION TITLE
Available for Italian 🍝

### DISCUSSION THREAD

**Discussion by ntotao** [anyproto/contributors#26] [Localization]
I'm available to contribute to the italian translation / check on whatever platform for localization will be used.

**Comment by fuksman** [comment]
Hey @ntotao, thank you for the desire to contribute. We are polishing localization files and hopefully will launch the l18n project next week. I think we will use [https://crowdin.com](https://crowdin.com) as a platform. If you have any other suggestions for this, please share them!

**Comment by ntotao** [comment]
great! I have much experience with crowdin - super easy to use - looking forward to it!

**Reply by fuksman** [reply]
Hey @ntotao, the l10n project is not launched officially, but you can already find it on [Crowdin](https://crowdin.com/project/anytype-desktop). While it's safe to start providing translations, we haven't finished extracting strings yet, so new strings will appear eventually.
